test/storageTests.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env mocha */
+import chai from "chai"
+import fs from 'fs'
+
+import createStorage from "../src/modules/storage"
+
+const expect = chai.expect;
+
+describe("Test storage", function () {
+  let storage = createStorage("tests")
+  fs.existsSync('data/tests.json') && fs.unlinkSync('data/tests.json')
+  fs.existsSync('data/tests-other.json') && fs.unlinkSync('data/tests-other.json')
+  fs.existsSync('data/tests-substore.json') && fs.unlinkSync('data/tests-substore.json')
+  it("default store", function () {
+    let store = storage.createStore() // default
+    expect(store.value).to.equal(undefined)
+    expect(store).to.equal(storage._getCurrentStores()['data/tests.json'])
+    store.close()
+    expect(storage._getCurrentStores()['data/tests.json']).to.equal(undefined)
+  })
+  it("default store with value", function () {
+    let store = storage.createStore(false, "defaultValue")
```

Comment:
nit (opcional): ese `false` como primer parametro es rarisimo.  No te conviene usar un valor por defecto y usar los parametros invertidos?

```
storage.createStore("defaultValue"); // asume  ''
```

Reply:
Sí, eso no me cerraba, pero quería que quede pegado al `createStore` para que se note más que es el nombre del store. ¿Algo así qué te parece?
```
createStore()("defaultValue")
createStore("substore")("otroDefaultValue")
```

Reply:
Decia algo como en lugar de:

```suggestion
  createStore: (root, defaultValue) => {
    const suffix = root ? `-${root}` : ''
```

hacer algo asi:

```suggestion
  createStore: (defaultValue, root = '') => {
    const suffix = root ? `-${root}` 
```

__creo__ me parece raro que `root` se pueda usar como booleano  o string.

Reply:
Es más bien un opcional pero poner `undefined` me dio flaca.